gravitee-apim-e2e/README.md

```diff
@@ -14,8 +14,11 @@ npm install
 ```
 
 Then, the following NPM scripts are available:
- - `npm run test`: Opens the Cypress Test Runner to run tests against a locally running APIM Rest API
- - `npm run test:api:management`: Run end to end tests of management rest-api against a locally running APIM.
+ - `npm run test:ui`: Run the cypress UI tests against a locally running APIM
+ - `npm run test:ui:docker`: Run the APIM stack and the cypress UI tests in docker ([more details](./docker/ui-tests/README.md))
+ - `npm run test:api`: Run the jest API tests against a locally running APIM
+ - `npm run test:api:docker`: Run the APIM stack and the jest API tests in docker ([more details](./docker/api-tests/README.md))
+ - `npm run test:serve:apim`: Run the APIM stack in docker in order to run tests locally from your IDE
```

Comment:
I would prefer to see the default `test:ui` behavior start docker and have a command like `test:ui:dev` for a locally apim

Reply:
and I will rename `test:serve:apim` to `serve:apim`, this command doesn't run tests